README.md

```diff
@@ -20,7 +20,7 @@ Minor modifications were made in order to get sct to:
 Compile the code using the following command:
 
 ~~~
-cc -std=c99 -O2 -I /usr/X11R6/include sct.c -o xsct -L /usr/X11R6/lib -lX11 -lXrandr
+gcc -Wall -std=c99 -O2 -I /usr/X11R6/include sct.c -o xsct -L /usr/X11R6/lib -lX11 -lXrandr -lm -s
```

Comment:
```suggestion
gcc -Wall -Wextra -Werror -pedantic -std=c99 -O2 -I /usr/X11R6/include sct.c -o xsct -L /usr/X11R6/lib -lX11 -lXrandr -lm -s
```
we might want to do that

Reply:
And in the Makefile?

Reply:
https://github.com/faf0/sct/pull/10/commits/4c2f7a1390a0fba58ea52632504122cb3a20bbd6